hasura/migrations/default/1650588140371_add_users_profile_exists_trigger/up.sql

```diff
@@ -1,7 +1,7 @@
 CREATE OR REPLACE FUNCTION function_ensure_profile_exists() RETURNS TRIGGER AS
 $BODY$
   BEGIN
-      INSERT INTO "profiles"(address) VALUES(new.address) ON CONFLICT DO NOTHING;
+      INSERT INTO "profiles"(address, name) VALUES(new.address, new.name) ON CONFLICT DO NOTHING;
```

Comment:
@sshmm i think we need this code to be in a new migration, because old migrations are not re-run.... right?